scripts/check_style.sh

```diff
@@ -1,9 +1,11 @@
 #!/usr/bin/env bash
 
-REPO_ROOT="$(dirname "$0")"/..
-
 (
+set -e
```

Comment:
Shouldn't we remove this now? This may cause the script to fail, if the greps fail? And in the interesting failure cases we have an explicit ``exit 1``, which should propagate from the subshell to the entire script, since it's only one subshell now.

Reply:
the greps are executed in a subshell of this subshell